src/MeoAssistant/CreditShoppingTask.cpp

```diff
@@ -122,8 +122,8 @@ bool asst::CreditShoppingTask::credit_shopping(bool white_list_enabled, bool cre
         sleep(rare_delay);
 
         if (credit_ocr_enabled) {
-            std::string credit = credit_ocr();
-            if (credit == "" || std::stoi(credit) <= m_max_credit) {//信用值不再溢出，停止购物
+            int credit = credit_ocr();
+            if (credit <0 || credit <= m_max_credit) {//信用值不再溢出，停止购物
```

Comment:
```suggestion
            if (credit <= m_max_credit) {//信用值不再溢出，停止购物
```

下面的 `credit > 0 && ...` 也可以改